javaScript/popup.js

```diff
@@ -0,0 +1,301 @@
+const navHead = document.querySelector('nav');
```

Comment:
- [ ]  OPTIONAL,  Kindly fix the overflow on the popup window so that it can be positioned nicely.